src/app/features/cases/components/organisms/Workflow/WorkflowStatus.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+import { Heading, themeColor, themeSpacing } from "@datapunt/asc-ui"
+import styled from "styled-components"
+
+import Table from "app/features/shared/components/molecules/Table/Table"
+
+type CellContent = React.ReactNode
+
+type Props = {
+  status: string    
+  data: Array<Array<CellContent>>
+  showBWVMessage?: boolean
+}
+
+const NewStyledTable = styled(Table)`
+  line-height: ${ themeSpacing(12) };
+  margin-bottom: ${ themeSpacing(5) };
+  th {
+    color: ${ themeColor("tint", "level4") };
+    border-color: ${ themeColor("tint", "level4") };
+  }
+`
+
+const StyledHeading = styled(Heading)`
+  margin-bottom: ${ themeSpacing(5) };
+`
+
+const Cap = styled.span`
```

Comment:
Capitalized or UPPERCASE?

Reply:
I changed it later to uppercase.. You mean the name of the const doesn't match it's function?